aws/resource_aws_client_vpn_endpoint_test.go

```diff
@@ -0,0 +1,259 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAwsClientVpnEndpoint_importBasic(t *testing.T) {
```

Comment:
Nit: New resources are normally written without a specific import test like this (the ImportState step is just added to all other test cases instead)

Reply:
I have removed this test and rolled the functions in the import file into the main resource file.